### ISSUE TITLE
Unable to handle NewWindowRequested 

### ISSUE THREAD

**Issue by MattChique** [MicrosoftEdge/WebView2Feedback#2314]
**Description**
Since last or last but one version of WebView2, we are not able to prevent opening a new window, when dragging in a file (for example a pdf file).

```csharp
coreWebView.NewWindowRequested += (_, args) =>
{
    args.Handled = true;
};
```
This is how we did it and what should work. Instead of suppressing it with handled=true, the file opens in a new window.

**Version**
6.0.200-preview.12.2441 worked fine, current 6.0.200-preview.14.5099 doesn't

[AB#38745753](https://microsoft.visualstudio.com/90b2a23c-cab8-4e7c-90e7-a977f32c1f5d/_workitems/edit/38745753)

**Comment by champnic** [comment]
Hey @MattChique - thanks for the bug report, and sorry you are running into this. To confirm, this used to work correctly and recently regressed? I've added this bug to our backlog to investigate.

**Comment by champnic** [comment]
Does this happen with all new windows, or just when navigating to a file? Can you confirm that the function is being hit?

**Comment by MattChique** [comment]
Hej @champnic yes, it used to work correctly and regressed recently. It worked fine in 6.0.200-preview.12.2441. In current 6.0.200-preview.14.5099 it doesn't.

The function is hit, but setting `args.Handled = true;` is not recognized. 
The window opens, regardless if dragging in a file or calling something like `await coreWebView.AddScriptToExecuteOnDocumentCreatedAsync("window.open('https://www.google.de')");`

**Comment by champnic** [comment]
Thanks for confirming!

What product do your version numbers correspond to? Typically I would expect to see "9x.0.1xxx.xx" for a WebView2 runtime version, or "1.0.1xxx.xx" for a WebView2 SDK version.

**Comment by MattChique** [comment]
Hej @champnic thanks for your quick responses.

Sorry, I should have mentioned the belonging of the version. 

6.0.200-preview.14.5099 is the version of the nuget package, which contains WebView2 1.0.1020.30.

![image](https://user-images.githubusercontent.com/26269158/160989298-d8623321-7597-444b-93b7-ec5047745b54.png)

**Comment by Flaflo** [comment]
Opening popup windows through JS dont work either. ``window.open("about:blank")`` tries to open a new process and windows asks to search for an appropriate software in the microsoft store.

**Comment by Flaflo** [comment]
@champnic I managed to find out why this is happening. This is not a fault from WebView2. Its rather maui that registered an event that manually handles new window requests and opens an external process.

I opened up an Issue in their repository. I think this can be closed. Thank you for your time! :)

Link to the issue in MAUI (I've included a workaround aswell):
https://github.com/dotnet/maui/issues/7930
